shell/client/admin-client.js

```diff
@@ -760,13 +790,11 @@ Template.adminAdvanced.events({
     const state = Iron.controller().state;
     const token = this.token;
     resetResult(state);
-    state.set("numSettings", 12);
+    state.set("numSettings", 11);
```

Comment:
Just skimming this review, I have no idea as a reader why a seemingly-magic number of 12 changed to a different seemingly-magic number of 11.